Source/Target.swift

```diff
@@ -49,14 +49,23 @@ public enum StructTarget: TargetType {
 
 /// Represents an HTTP method.
 public enum Method: String {
-    case GET, POST, PUT, DELETE, OPTIONS, HEAD, PATCH, TRACE, CONNECT
+
+    case options = "OPTIONS"
```

Comment:
Tried this and seems to work fine.

```swift
public typealias Method = Alamofire.HTTPMethod

extension Method {
    public var supportsMultipart: Bool {
        switch self {
        case .post,
             .put,
             .patch,
             .connect:
            return true
        default:
            return false
        }
    }
}
``` 

Only downside I can see here is that it's yet another dependency on Alamofire. But given the discussion in #556, not sure if that's an actual issue.